go.mod

```diff
@@ -53,13 +53,13 @@ require (
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 
 	//pinned k8s.io to v0.24.1 tag
-	k8s.io/api v0.24.4
+	k8s.io/api v0.25.1
```

Comment:
@PrasadG193 Don't of these need to be upgrade in tandem? including 
`k8s.io/apiextensions-apiserver v0.24.4`
`k8s.io/kubectl v0.24.4`
and potentially 
`sigs.k8s.io/controller-runtime v0.12.3`